docs/index.html

```diff
@@ -0,0 +1,766 @@
+<html> 
```

Comment:
Please don't checkin HTML in this repository, especially if it should be autogenerated.

We can include those in #209 for the documentation.

Reply:
removed HTML. It was just there to simplify review of the text; although the help text conformed to the markdown used by txt2man(1) so that  manpages and HTML can be automatically generated; as I installed cargo(1) for comparisons and noted that all the documentation is available via a help subcommand in cargo(1) and as manpages. Had that in mind. Normally i would use ufpp(1) to put the help in a text block in the code and use it to automatically change the text block to a variable declaration and generate an input file for txt2man(1) and man2html(1) or groff(1) so you get all three (help text, manpage, HTML page) from a simple in-line text block in the source file; so never really got to interested in FORD or doxygen except that I find the graphs from doxygen interesting. How to I look at the FORD output that #209 infers is generated? I could not find it.